soc/telink/tlsr/tlsr951x/Kconfig.soc

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2021 Telink Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_TLSR951X
+	bool
+	select SOC_FAMILY_TELINK_TLSR
```

Comment:
shouldn't selects go into `Kconfig`?

Reply:
Not for family no, that is the exception